int32 duty_cycle_l
int32 duty_cycle_r
